packages/testcontainers/src/containers/RegTestContainer/index.ts

```diff
@@ -55,7 +56,7 @@ export class RegTestContainer extends DeFiDContainer {
     return cmds
   }
 
-  async getNewAddress (label: string = '', addressType: 'legacy' | 'p2sh-segwit' | 'bech32' | string = 'bech32'): Promise<string> {
+  async getNewAddress (label: string = '', addressType: 'legacy' | 'p2sh-segwit' | 'bech32' | 'eth' | string = 'bech32'): Promise<string> {
```

Comment:
Actually, you may start using `NativeChainContainer`.

https://github.com/BirthdayResearch/jellyfishsdk/blob/afed8a45e3a7b768478ff368b695ceea0c44201a/packages/testcontainers/src/containers/NativeChainRpc.ts#L120C7-L122

Refer to the tests for how to use / implement:
https://github.com/BirthdayResearch/jellyfishsdk/blob/main/packages/testcontainers/__tests__/containers/NativeChainContainer/Masternode.test.ts